services/gitea/tasks/create_user.yml

```diff
@@ -0,0 +1,64 @@
+- name: "Create Gitea user(s)"
+  uri:
+    url: '{{ gitea_api_url }}/admin/users'
+    method: POST
+    body_format: json
+    status_code: 201
+    force_basic_auth: yes
+    url_username: '{{ gitea_admin_user }}'
+    url_password: '{{ gitea_admin_passwd }}'
+    body:
+      username: '{{ user.name }}'
+      password: '{{ user.password }}'
+      email: '{{ user.email }}'
+      admin: '{{ "true" if user.admin else false }}'
+      must_change_password: false
+    return_content: true
```

Comment:
this needed?

Reply:
was used for testing, will be removed in the next commit

Reply:
seems like it's still there